modules/core/src/lib/deck.js

```diff
@@ -21,8 +21,9 @@
 import LayerManager from './layer-manager';
 import ViewManager from './view-manager';
 import MapView from '../views/map-view';
-import EffectManager from '../experimental/lib/effect-manager';
-import Effect from '../experimental/lib/effect';
+import EffectManager from '../effects/effect-manager';
```

Comment:
Same for EffectManager, I would move it to `lib`. 

If the lib folder is getting too big, we could think about how to create subfolders, e.g. `lib/effect`.